### PR TITLE
Percy snapshot test

### PR DESCRIPTION
**Pull request by philrenaud**
(test runner PR)

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -243,7 +243,7 @@ jobs:
           path: c:\tmp\test-reports
   test-ui:
     docker:
-      - image: docker.mirror.hashicorp.services/circleci/node:12-browsers
+      - image: docker.mirror.hashicorp.services/circleci/node:14-browsers
```
Comment (ChaiWithJai) [inline .circleci/config.yml:246]: `question`: should we review the changelog in Node to are deprecations or breaking changes that we should consider?

What's the surface area of this change?

File: ui/tests/acceptance/volumes-list-test.js [hunk 2]
```diff
@@ -24,7 +26,7 @@ module('Acceptance | volumes list', function (hooks) {
   setupApplicationTest(hooks);
   setupMirage(hooks);
 
-  hooks.beforeEach(function () {
+  hooks.beforeEach(function (hook) {
```
[the remark this answers is not in the document]
Comment (philrenaud) [reply ui/tests/acceptance/volumes-list-test.js]: Great catch; removed.

File: ui/tests/acceptance/clients-list-test.js [hunk 3]
```diff
@@ -28,12 +30,14 @@ module('Acceptance | clients list', function (hooks) {
   test('/clients should list one page of clients', async function (assert) {
     // Make sure to make more nodes than 1 page to assert that pagination is working
     const nodesCount = ClientsList.pageSize + 1;
-
+    faker.seed(1); // stabilize percy
```
[the remark this answers is not in the document]
Comment (philrenaud) [reply ui/tests/acceptance/clients-list-test.js:33]: Percy does visual-diff checking, which means pixel-by-pixel comparisons. We're mostly looking for CSS or model changes that have an impact on what the user sees.

Our mirage tests generate random data by default — different for every test run. This is useful for helping us catch edge cases ("this line wraps under some circumstances but not under others", etc.) but for percy, it means that every single test run would result in a visual diff. Not what we want.

Thus, we stabilize it with faker.seed(N) to ensure the data stays the same for these tests.

File: .circleci/config.yml (hunk 1 above)
Comment (lgfa29) [reply .circleci/config.yml:246]: I think we're good here. I took a look at it quickly a while back and didn't see anything relevant. We also use node as a built tool, not the runtime, so any breaking change would likely cause the UI to not even build.

**Review by lgfa29 — APPROVED** [review]
Nice one! This will be super helpful.

**Pull request closed without merging**